src/test/component/test_component.py

```diff
@@ -23,6 +25,36 @@ def serverless_yaml():
     )
 
 
+@pytest.fixture()
```

Comment:
These are component tests.
I do want that they will interact with the internet.
Is there any particular reason for this change?

Reply:
Yes, httpbin is flaky and returns gateway timeout errors. This solution creates an HTTP server that runs on another port on localhost, that any http library can communicate with

Reply:
ok then 👍